visualizations/frontend/filters/v2/TMultiSelector.vue

```diff
@@ -1,110 +1,115 @@
-					:class="checked.length ? 'text-[#145DEB]' : 'text-[#727184]'"
-				>
-					<t-loading-spinner v-if="isExpanded && loading" position="relative" />
-					<span v-else>
-						{{ checked.length < ids.length ? 'Select All' : 'Reset' }}
-					</span>
-				</span>
-			</div>
-			<span
-				v-else
-				@click="selectUnselect"
-				class="text-[#145DEB] text-[13px] cursor-pointer font-normal leading-[18px] inline-flex items-center justify-end px-4 grow-1"
-				:class="checked.length ? 'text-[#145DEB]' : 'text-[#727184]'"
-			>
-					<t-loading-spinner v-if="isExpanded && loading" position="relative" />
-					<span v-else>
-						{{ checked.length < ids.length ? 'Select All' : 'Reset' }}
-					</span>
-				</span>
-			<div class="px-[8px] pt-[4px] pb-[6px] w-full">
-				<virtual-list 
-					:style="{
-						height: Math.min(menu.length * 30, 320) + 'px',
-						'overflow-y': 'auto'
-					}"
-					:estimate-size="32"
-					:extra-props="{checked, updateUrlParam}"
-					:data-key="'index'"
-					:data-sources="menu"
-					:data-component="itemComponent"
-				/>
-				<small v-if="!menu.length && !loading">
-					No items found
-				</small>
-			</div>
-		</div>
-	</t-dropdown>
+    <!-- Popup Contents -->
+    <div class="min-w-[294px]" v-bind="popupAttrs">
+      <div
+        v-if="label"
+        class="px-[12px] pt-[16px] pb-[8px] flex justify-between items-center w-full"
+      >
+        <h6
+          class="text-[10px] text-[#727184] font-semibold uppercase leading-[15px] tracking-widest"
+        >
+          {{ label }}
+        </h6>
+      </div>
+      <div class="px-2 py-2 flex nav-search" v-if="isSearchable">
+        <div class="grow-2">
+          <base-search-input
+            class="mt-0 text-sm search-report !rounded-md"
+            :placeholder="searchPlaceholder"
+            size="small"
+            :clearable="false"
+            v-model="search"
+          />
+        </div>
+        <span
+          @click="selectUnselect"
+          class="text-[#145DEB] text-[13px] cursor-pointer font-normal leading-[18px] inline-flex items-center justify-end px-4 grow-1"
+          :class="checked.length ? 'text-[#145DEB]' : 'text-[#727184]'"
+        >
+          <t-loading-spinner v-if="isExpanded && loading" position="relative" />
+          <span v-else>
+            {{ checked.length < menu.length ? "Select All" : "Reset" }}
+          </span>
+        </span>
+      </div>
+      <span
+        v-else
+        @click="selectUnselect"
+        class="text-[#145DEB] text-[13px] cursor-pointer font-normal leading-[18px] inline-flex items-center justify-end px-4 grow-1"
+        :class="checked.length ? 'text-[#145DEB]' : 'text-[#727184]'"
+      >
+        <t-loading-spinner v-if="isExpanded && loading" position="relative" />
+        <span v-else>
+          {{ checked.length < menu.length ? "Select All" : "Reset" }}
```

Comment:
Prev statement
`{{ checked.length < ids.length ? "Select All" : "Reset" }}`

Problem was, menu is reactive and changes with total visible items while ids stay constant, using ids for comparison results in `Reset` never showing.